shared/xccdf/services/base.xml

```diff
@@ -117,7 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
-<platform idref="cpe:/a:machine"/>
+<machine />
```

Comment:
How about something like `<platform-restriction/>` which defaults to machine=true, container=true, vm=true and you can override any of them?

`<platform-restriction container="false"/>`

It's not great and something even a little better is required but it communications that there is some restriction on where this rule is to be used. And `platform-restriction` is not the right name for this element.